README.md

```diff
@@ -1,2 +1,228 @@
 # @logdna/env-config
 
+> Node.js Package to define, document, and assert environment variables
+
+## Install
+
+```
+$ npm install --save @logdna/env-config
+```
+
+## Test
+
+```
+$ npm t
+```
+
+## Usage
+
+The recommended way to structure applications with this package is as follows:
+
+```js
+// config.js
+
+'use strict'
+
+const Config = require('@logdna/env-config')
+const config = new Config([
+  Config.string('loglevel').default('info')
+, Config.number('port').default(3000)
+])
+
+module.exports = config
+```
+
+```js
+// index.js
+//
+// The following env vars can be set:
+//
+// LOGLEVEL - defaults to info
+// PORT - default to 3000
+//
+
+'use strict'
+
+const http = require('http')
+const config = require('./config.js')
+
+// This validates that we have the necessary env vars.
+config.validateEnvVars()
+
+http.listen(config.get('port'), () => {
+  log.info('listen', config.get('port'))
+})
+```
+
+Under the hood, `Config` is a [`<Map>`][], so use it like one.
+
+This package also provides a way to automatically generate documentation
+for the environment variables for a service.
+If the `doc` directory does not exist, it can be created using the following:
```

Comment:
think we could be consistent about line-length

Reply:
Consistent how?  These are all under 80 characters.

Reply:
in the rendered markdown, these two lines don't render a line break... 

<img width="948" alt="image" src="https://user-images.githubusercontent.com/33665323/94852691-45a2c500-03f8-11eb-81fc-112e56c1de84.png">

Reply:
Gotcha.  I broke up the line into paragraphs.